vignettes/Treatment.Rmd

```diff
@@ -15,50 +15,188 @@ knitr::opts_chunk$set(
 ```
 
 ```{r setup}
-suppressPackageStartupMessages(library(ggplot2))
 library(malariasimulation)
 library(malariaEquilibrium)
```

Comment:
Is it necesary to load malariaEquilibrium? I think any malariasimulation functions that depend on it should work without this?

Reply:
I've removed this from the code.